dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -10,8 +10,10 @@ def index
     render json: @workshops, each_serializer: Api::V1::Pd::WorkshopSerializer
   end
 
+  # Upcoming (not started) public CSF workshops.
   def k5_public_map_index
     @workshops = Pd::Workshop.where(
+      started_at: nil,
```

Comment:
I'll also add a date filter in case a workshop is never started or deleted, but still in the past.